openshift/template.yaml

```diff
@@ -521,7 +521,9 @@ objects:
                     domains: ["*"]
                     routes:
                     - match: { prefix: "/" }
-                      route: { cluster: assisted_service }
+                      route:
+                        cluster: assisted_service
+                        timeout: 180s
```

Comment:
I understand this is setting the timeout the same as OCM gateway, but why is it so high?
Nevertheless I think making timeout explicit it's a good thing, could we parametrize it though?